packages/docusaurus-1.x/lib/server/liveReloadServer.js

```diff
@@ -29,7 +29,9 @@ function start(port) {
 
 const getReloadScriptUrl = () => {
   const port = process.env.LIVERELOAD_PORT;
-  return `http://localhost:${port}/livereload.js`;
+  const server = process.platform === 'win32' ? 'localhost' : '0.0.0.0';
```

Comment:
Nit - It's more of `host` than a `server`. Same issue regarding OS as above.

Reply:
Ill make this adjustment with a commit alongside the change for https://github.com/facebook/Docusaurus/pull/1475#discussion_r284958730